sdk/servicebus/service-bus/package.json

```diff
@@ -94,23 +107,25 @@
     "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
     "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
     "build:samples": "node .scripts/prepSamples.js && cd samples && tsc -p .",
-    "build:test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test:browser": "tsc -p . && cross-env BROWSER_TEST=true rollup -c 2>&1",
+    "build:test:node": "tsc -p . && rollup -c rollup.test.config.js 2>&1",
```

Comment:
I notice the main difference here is that you're no longer prefixing `rollup` with `cross-env ONLY_NODE=true` - is that change intentional?

Reply:
Yes, it's because the test rollup script does not use this variable as such

Reply:
👍